# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/notifications/registration-association.https.window.js.ini
# Commit: 2e24d997d5de
# Full Hash: 2e24d997d5de727be8e44b522e57d7ff7db52486
# Author: Sandor Molnar <smolnar@mozilla.com>
# Date: 2025-07-16 21:45:35
# Regressor Bug: 1972117
# File Overlap Count: 1
# Description:
#   Revert "Bug 1972117: apply code formatting via Lando" for causing assertion failures @ MozPromise.h
#   
#   This reverts commit e76333b4d7fb2d323e797f12bde50ccc99a51a85.
#   
#   Revert "Bug 1972117 - Part 4: Clear notifications on service worker unregistration r=asuth"
# ==============================================================================

diff -r c257659aa2b4 -r 2e24d997d5de testing/web-platform/meta/notifications/registration-association.https.window.js.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/meta/notifications/registration-association.https.window.js.ini	Wed Jul 16 19:18:03 2025 +0300
@@ -0,0 +1,9 @@
+[registration-association.https.window.html]
+  [A new SW registration gets no previous notification]
+    expected: FAIL
+
+  [An unregistered SW registration gets no previous notification]
+    expected:
+      if (os == "linux") and fission and asan: [PASS, FAIL]
+      if (os == "android") and debug and not sessionHistoryInParent: [PASS, FAIL]
+      if (os == "linux") and not fission: [PASS, FAIL]
